app/database/queries/breeze_buddy/template.py

```
@@ -419,3 +420,184 @@ def replace_template_query(
+
+
+def assign_family_query(
+    family_id: str, template_ids: List[str], now
+) -> Tuple[str, List[Any]]:
+    query = f"""
+        UPDATE {TEMPLATE_TABLE}
+        SET family_id = $1,
+            updated_at = $2
+        WHERE id = ANY($3::uuid[])
+        RETURNING id
+    """
```

Ruff (0.16.1)

[error] Possible SQL injection vector through string-based query construction

(S608)